tools/docker/scripts/keycloak-setup.sh

```diff
@@ -17,7 +18,5 @@ ansible-galaxy collection build tools/keycloak_setup/collection
 echo -e "\e[34m >>> Installing keycloak setup collections \e[97m"
 ansible-galaxy collection install community.general pinakes-keycloak_setup-"$KEYCLOAK_SETUP_VERSION".tar.gz
 echo -e "\e[34m >>> Configuring Keycloak \e[97m"
-ansible-playbook tools/keycloak_setup/dev.yml -vvv
-
-ansible-playbook -vvv tools/keycloak_setup/dev.yml
```

Comment:
👍 I was wondering why the keycloak setup was slow, should we also get rid of the vvv @Alex-Izquierdo

Reply:
mmm I think the verbose is useful for debugging if there is some problem

Reply:
Hi @mkanoor I've added the debug optional, but it not affects to the performance, just prints more output messages.